util/devel/chplspell-dictionary

```diff
@@ -5645,6 +5670,7 @@ malayalam
 malloc
 maltzahn
 mandelbrot
+mannari
```

Comment:
Where does this one come from?

Reply:
A new entry in the contributors file.

Reply:
Cool, that makes sense (and was kinda what I was guessing)